README.md

```diff
@@ -63,7 +63,8 @@ In case of replicating the analysis without using `conda`, the following are the
   - python-graphviz
   - pip
   - rpy2
-  - jupyter-book
```

Comment:
Seems that some of my changes were reverted as the base in your repo wasn't the latest one.
Next time you can make a pull before PR (it needs practice... no worries)

So for this file,
keep -jupyter-book, -dataframe-image, and remove -kaggle